rewriter/tests/global_fn_ptr/operations.c

```diff
@@ -2,18 +2,17 @@
 RUN: cat global_fn_ptr_call_gates_1.ld | FileCheck --check-prefix=LINKARGS %s
 */
 #include "operations.h"
+#include <criterion/criterion.h>
 #include <stdio.h>
 
 extern Op operations[2];
 
-// LINKARGS: --wrap=call_operations
-void call_operations(void) {
-    for (int i = 0; i < 2; i++) {
-        // TODO: Add a way to share strings between compartments
-        //printf("%s\n", operations[i].desc.data);
-        uint32_t x = 18923;
-        uint32_t y = 24389;
-        uint32_t res = operations[i].function(x, y);
-        printf("%d = f(%d, %d)\n", res, x, y);
-    }
+// LINKARGS: --wrap=call_operation
+uint32_t call_operation(size_t i) {
+    // TODO: Add a way to share strings between compartments
+    //printf("%s\n", operations[i].desc.data);
```

Comment:
nit: I think we can remove the comment on the printf and the TODO We previously treated the `.rodata` sections (where string literals are placed) as secret, but we have since moved on to only treating writable sections that way.